sdccc/src/main/java/com/draeger/medical/sdccc/configuration/DefaultTestSuiteModule.java

```diff
@@ -51,4 +55,17 @@ protected void configure() {
         bind(LocalAddressResolver.class).to(LocalAddressResolverImpl.class).in(Singleton.class);
         bind(ManipulationSerializer.class).to(GsonManipulationSerializer.class).in(Singleton.class);
     }
+
+    /**
+     * Provides a singleton Gson instance configured with a custom OptionalTypeAdapter.
+     * @return A configured, singleton Gson instance.
+     */
+    @Provides
+    @Singleton
+    public Gson provideGson() {
```

Comment:
How are users supposed to extend this with other type adapters without building a second instance?